easybuild/easyconfigs/g/GROMACS/GROMACS-2016.3-CrayGNU-17.08-PLUMED-2.3.0-cuda-8.0.61.eb

```diff
@@ -0,0 +1,46 @@
+# contributed by Luca Marsella (CSCS)
+#
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+#
+name = 'GROMACS'
+version = "2016.3"
+cudaversion = '8.0'
+versionsuffix = '-PLUMED-2.3.0-cuda-%s' % cudaversion
```

Comment:
cudaversion is `8.0`, so could you please rename the file in order to match that? (remove the `.61`)

Reply:
Done.